cristin-import/src/main/java/no/unit/nva/cristin/mapper/nva/ReferenceBuilder.java

```diff
@@ -194,7 +194,7 @@ private PublicationContext buildEventForPublicationContext() {
     private Time extractTime() {
         var from = extractCristinEventFromDate();
         var to = extractCristinEventToDate();
-        return nonNull(from) && nonNull(to) ? new Period(from, to) : null;
+        return nonNull(from) || nonNull(to) ? new Period(from, to) : null;
```

Comment:
What is the expected behavior when the "from" date is missing? Does that make sense or would that be an error in the data?

Reply:
One of dates can be missing, we are migrating data from Cristin and we have to import dates/periods as is. It will not be an error in the data.

Reply:
It can be the case where user has started to create draft publication, but is not sure when event actually starts, and leaves this field empty.

Reply:
Aha, I see!